src/sentry/tasks/sentry_apps.py

```diff
@@ -239,8 +239,30 @@ def installation_webhook(installation_id, user_id, *args, **kwargs):
 @instrumented_task(name="sentry.tasks.sentry_apps.workflow_notification", **TASK_OPTIONS)
 @retry(**RETRY_OPTIONS)
 def workflow_notification(installation_id, issue_id, type, user_id, *args, **kwargs):
-    extra = {"installation_id": installation_id, "issue_id": issue_id}
+    install, issue, user = get_webhook_data(installation_id, issue_id, user_id)
 
+    data = kwargs.get("data", {})
+    data.update({"issue": serialize(issue)})
+    send_webhooks(installation=install, event=f"issue.{type}", data=data, actor=user)
+
+
+@instrumented_task(name="sentry.tasks.sentry_apps.build_comment_webhook", **TASK_OPTIONS)
+@retry(**RETRY_OPTIONS)
+def build_comment_webhook(installation_id, issue_id, type, user_id, *args, **kwargs):
+    install, _, user = get_webhook_data(installation_id, issue_id, user_id)
+    data = kwargs.get("data", {})
+    payload = {
+        "comment_id": data.id,
+        "group_id": data.group.id,
+        "project_slug": data.project.slug,
+        "timestamp": data.datetime,
+        "comment": data.data["text"],
```

Comment:
Do external integrations need the user_id or `actor` payload to do their sync work?

Reply:
Oh hmm that's a good point, a little hard to say since obviously nobody is using it yet :p but if they want to attribute the comment to the user who posted it yeah that makes sense, I'll add it.

Reply:
Wait a second we already have the actor stuff in there lol, it's just not added at this time.

Reply:
do we need to provide any additional data for tagged teams/users in a comment?

Reply:
I don't think so, I left a comment tagging both a user and a team and we store it like this:

```
    "data": {
      "text": "hello **@aniket.das@sentry.io** sorry ignore this is a test about tags **#hellboy**"
    },
```

Reply:
I guess if we wanted to be really fancy we could parse out the stuff between the double asterisks and lookup the actor data based on it, but I don't know if that's necessary.

Reply:
I think it might be useful, but for mvp we don't need it. I think we should wait for it being asked as a feature request by integration partners.

Reply:
> Wait a second we already have the actor stuff in there lol, it's just not added at this point in the code. If you look at the payloads I put in the description it's in there.

Sorry about that, I should have looked at the tests.